### PR TITLE
fix(planning): disable planning module

### PR DESCRIPTION
**Pull request by petrjasek**
disable planning module not to confuse users

SD-3464

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

File: client/app/index.html [hunk 1]
```diff
@@ -143,7 +142,6 @@
             'main',
             'superdesk-settings/module',
             'superdesk-dashboard/module',
-            'superdesk-planning/module',
             'superdesk-archive/module',
             'superdesk-ingest/module'
             //'superdesk-scratchpad/module',
```
Comment (plamut) [inline client/app/index.html]: Should this be removed entirely?

**Comment by plamut** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**